packages/framework/src/Model/Customer/User/Role/CustomerUserRole.php

```diff
@@ -9,4 +9,15 @@ class CustomerUserRole
     public const ROLE_API_LOGGED_CUSTOMER = 'ROLE_API_LOGGED_CUSTOMER';
     public const ROLE_API_ALL = 'ROLE_API_ALL';
     public const ROLE_API_CUSTOMER_SELF_MANAGE = 'ROLE_API_CUSTOMER_SELF_MANAGE';
+
+    /**
+     * @return array<string, string>
```

Comment:
does it make sense to extend the `Shopsys\FrameworkBundle\Component\Enum\AbstractEnum` for enum-like behavior?

Reply:
I do not see any advantage of extending AbstractEnum. What shall I use it for ?

Reply:
We could easily get all cases, but it's not necessary in this scenario